### PR TITLE
Add: Feedback Modal

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by firefinchdev** [comment]
@witty123 
@abhishek97 
Pls Review
Solves #253 .

**Comment by firefinchdev** [comment]
Motley changes
https://github.com/coding-blocks/motley/pull/270

**Comment by abhishek97** [comment]
Why you copied the rating-component? why can't it be reused here?

**Comment by firefinchdev** [comment]
Ok, will update it to single component.

**Comment by firefinchdev** [comment]
@abhishek97 
Refactored to one component. 
Pls review.

**Comment by witty123** [comment]
@firefinchdev once the user has submitted his feedback, he should be able to edit it.

**Comment by firefinchdev** [comment]
screenshot from 2018-08-19 03-11-01
screenshot from 2018-08-19 03-11-03
Works perfectly on latest motley pull with #270

File: app/pods/components/rating-stars/component.js [hunk 1]
```diff
@@ -9,52 +9,84 @@ import { service } from 'ember-decorators/service'
...
+    this.get('api').request('/courses/' + this.get('course.id') + '/rating', {
+      method: 'POST',
+      data: {
+        value: this.get('ratingMarkedByUser'),
+        experience: this.get('textExperience'),
+        review: this.get('textPublic')
+      }
+    })
+    this.afterFeedback()
+    return true
```
Comment (abhishek97) [inline app/pods/components/rating-stars/component.js]: remove this

**Pull request merged**